docs/spec/spec.md

```diff
@@ -1,23 +1,20 @@
 # OrderlyWeb API
 
-Follows the general points in the [montagu api](https://github.com/vimc/montagu-api/blob/master/spec/spec.md)
+Follows the general points in the [montagu api](https://github.com/vimc/montagu-api/blob/master/docs/spec/spec.md)
```

Comment:
This link was broken

Reply:
Maybe we should remove this link and duplicate these "general points" in this spec, whatever they are! Does it just mean the below points or are there other points in the Montagu API spec that are missing here?

Reply:
Ah yeah, actually there is a list of general points, I'll copy that over